integration-tests/config/testplan.json

```diff
@@ -2,7 +2,7 @@
     "templates": ["go", "python", "nodejs", "dotnet-basic", "java-quarkus", "java-springboot"],
     "tssc": [{
       "git": "github",
-      "ci": "tekton",
+      "ci": "azure",
```

Comment:
Why this change? Is this something we want to keep going forward or just added to test the changes?